app/controllers/guides_controller.rb

```diff
@@ -30,7 +30,13 @@ def guide_params
                                   :location,
                                   :specialty,
                                   :rate,
-                                  :availability )
+                                  :sun_avail,
+                                  :mon_avail,
+                                  :tues_avail,
+                                  :wed_avail,
+                                  :thurs_avil,
```

Comment:
Oh yeah, here was the minor typo I mentioned in our meeting yesterday. Should be :thurs_avail not :thurs_avil